CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.24.3
+
+### Patch
+
+- Added C++ tokens
+
```

Comment:
Please revert this as well. I'll take care of the changelog before pushing a release.

Reply:
I have made changes requested, reverted changelog